copilot/fsd-form-runner/manifest.yml

```diff
@@ -10,21 +10,21 @@ http:
   # To match all requests you can use the "/" path.
   path: '/'
   # You can specify a custom health check path. The default is "/".
-  healthcheck: '/healthcheck'
-  alias: form-runner.test.access-funding.test.levellingup.gov.uk
+  healthcheck: '/health-check'
+  alias: forms.test.access-funding.test.levellingup.gov.uk
 
 # Configuration for your containers and service.
 image:
   # Docker build arguments. For additional overrides: https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#image-build
-  location: ghcr.io/communitiesuk/digital-form-builder-dluhc-runner:latest
+  location: ghcr.io/communitiesuk/runner:latest
```

Comment:
Is there a reason we can't use the existing `digital-form-builder-dluhc-runner` image?

Reply:
![image](https://github.com/communitiesuk/digital-form-builder/assets/125875751/e947825b-d959-4e83-a42b-effc3dafaa9a)
No particular reason at all, we can use any of the packages. But runner is the most up to date docker image, as runner has not been updated since August 2022

Reply:
the github sha on production tends to lend credence to the theory that it's runner, not digital-form-builder-dluhc-runner that is being used

cf env funding-service-design-form-runner 
...
GITHUB_SHA: runner:sha-3ab93d45b8054384f112a5799071774ee05d784b
...

https://github.com/communitiesuk/digital-form-builder/pkgs/container/runner/141861713?tag=sha-3ab93d45b8054384f112a5799071774ee05d784b

Reply:
Plus the current terraform utilises runner https://github.com/communitiesuk/funding-service-design-terraform-prod/blob/main/versions.tf#L10